src/py21cmfast/outputs.py

```diff
@@ -532,8 +540,9 @@ def _get_box_structures(self) -> Dict[str, Union[Dict, Tuple[int]]]:
             "xH_box": {"init": np.ones, "shape": shape},
             "Gamma12_box": shape,
             "MFP_box": shape,
+            "Nion_box": shape,
```

Comment:
@qyx268 I've just started reviewing this. This is probably why my tests fail for the branch. Should I change the tests accordingly then?

Reply:
I think the failing is due to the change of dNec_box to Nrec_box. I guess we will need to update the test files. Just ignore this for now. Andrei said I could make a PR for the lya develop branch. Once the branch is merged into master, you won't have the problem coming from my develop branch. You can of course change these things back to pass the test for now if you want.

Reply:
Okay, no problem then. I was just worrying why the tests failed, because it started when I added the test function, so I thought I made an error somewhere. This is fine then, but this red cross is really annoying haha